04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Jdaviz.ipynb

```diff
@@ -0,0 +1,474 @@
+    "* Plot spectra using the [Jdaviz](https://jdaviz.readthedocs.io/en/latest/index.html) data analysis visualization tool"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "934ba639-42a1-4649-a64b-25f8d33c9ad0",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"intro\"></a>\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook demonstrates how to find and retrieve spectroscopy data for certain objects from the DESI EDR data set using [SPARCL](https://astrosparcl.datalab.noirlab.eduhttps://astrosparcl.datalab.noirlab.edu) (SPectra Analysis and Retrievable Catalog Lab) and producing an interactive plot of an object's spectrum using [Jdaviz](https://jdaviz.readthedocs.io/en/latest/index.html).\n",
+    "\n",
+    "This notebook is based on [an example](https://github.com/camipacifici/jdaviz/blob/concept-sparcl/notebooks/concepts/specviz_sparcl.ipynb) originally developed by [Camilla Pacifici](https://github.com/camipacifici). Additional assistance was provided by\n",
+    "[Brett Morris](https://github.com/bmorris3). \n",
+    "\n",
+    "Packages:\n",
+    "\n",
+    "* [SPARCL](https://astrosparcl.datalab.noirlab.edu)\n",
+    "* [Jdaviz](https://jdaviz.readthedocs.io/en/latest/index.html)\n",
+    "* [jupyterlab-sidecar](https://github.com/jupyter-widgets/jupyterlab-sidecar)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc702b65-1e98-4b1f-9bc7-c5f2324eda40",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"disclaimer\"></a>\n",
+    "## Disclaimer & attribution\n",
+    "If you use this notebook for your published science, please acknowledge the following:\n",
+    "\n",
+    "* Data Lab concept paper: Fitzpatrick et al., \"The NOAO Data Laboratory: a conceptual overview\", SPIE, 9149, 2014, http://dx.doi.org/10.1117/12.2057445\n",
+    "\n",
+    "* Data Lab disclaimer: https://datalab.noirlab.edu/disclaimers.php\n",
+    "\n",
+    "* SPARCL: https://astrosparcl.datalab.noirlab.edu/sparc/acknowledgments/\n",
+    "\n",
+    "* Jdaviz: https://jdaviz.readthedocs.io/en/latest/index_citation.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4afe333-71b1-47a2-a0ca-2715a22ba771",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"imports\"></a>\n",
+    "## Imports and setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "1b12bf2e",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-10-10T19:54:13.337427Z",
+     "iopub.status.busy": "2023-10-10T19:54:13.336966Z",
+     "iopub.status.idle": "2023-10-10T19:54:18.272258Z",
+     "shell.execute_reply": "2023-10-10T19:54:18.271282Z",
+     "shell.execute_reply.started": "2023-10-10T19:54:13.337382Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# SPARCL import\n",
+    "from sparcl.client import SparclClient\n",
+    "# Jdaviz import\n",
+    "from jdaviz import Specviz\n",
```

Comment:
It seems that the module 'jdaviz' has not been installed by default for everyone. I am encountering the following error when running this cell: 'No module named 'jdaviz'.' Could you either make the package installation global or add instructions on how to install this module for individual accounts at the beginning of this NB?

Reply:
It should now be installed on both `gp02` and `gp12`.